package.json

```diff
@@ -2,26 +2,47 @@
   "name": "idbify",
   "version": "1.0.0",
   "description": "A simple interface for IDB with promises, you can define your schema up front and then it provides put, delete, and get functions.",
-  "main": "dist/idbify.es5.js",
-  "module": "dist/idbify.min.js",
+  "main": "dist/idbify.umd.js",
+  "module": "dist/idbify.module.js",
   "repository": "https://github.com/coltpini/idbify.git",
   "author": "Colt Pini <coltpini@github.com>",
   "license": "Apache 2.0",
+  "babel": {
+    "minified": true,
+    "compact": true,
+    "sourceMaps": true,
+    "presets": [
+      [
+        "es2015",
+        {
+          "modules": false
+        }
+      ],
+      "babili"
+    ],
+    "env": {
+      "umd": {
+        "plugins": [
+          "transform-es2015-modules-umd"
+        ]
+      }
+    }
+  },
   "scripts": {
     "test": "npm run pack && jest",
-    "pack": "npm run uglify && npm run babelify",
-    "uglify": "uglifyjs idbify.js -c -m -o dist/idbify.min.js --source-map",
-    "babelify": "babel idbify.js -o dist/idbify.es5.js --source-maps --presets=es2015 --minified --compact --plugins transform-es2015-modules-umd"
+    "prepare": "npm run build-module && npm run build-umd",
+    "build-module": "mkdir -p dist && babel idbify.js -o dist/idbify.module.js",
+    "build-umd": "mkdir -p dist && cross-env BABEL_ENV='umd' babel idbify.js -o dist/idbify.umd.js"
```

Comment:
Instead of configuring a babel `env` and having to pull in a dependency just to set an environment variable, would this work?

```sh
babel --plugins transform-es2015-modules-umd idbify.js ...
```